README.md

```diff
@@ -125,20 +122,20 @@ they follow inheritance tree as below:
 
 * `ReflectingUnixSocketEndpoint`, `ReflectingTCPIPEndpoint`: both of them send back
   the request data in the response body to the client for inspection. The only
-  difference between them is that the former is listening on Unix Socket
-  and the latter is listening on TCP/IP socket. They are useful for very simple
-  tests that only check if the request is hitting the right upstream and the
-  headers are correct.
+  difference between them is that the former is listening on Unix Socket and
+  the latter is listening on TCP/IP socket. They are useful for very simple
+  tests that only check if the request is hitting the right upstream and
+  the headers are correct.
 * `IAMEndpoint`, `MarathonEndpoint`, `MesosEndpoint`: specialized endpoints that
   are mimicking IAM, Marathon and Mesos respectively. Apart from the basic
   functionality (reset, bork, etc...), they are also capable of recording requests
   sent to them, and then returning them through mockers `send_command` back to the
   tests code.
 * all remaining endpoints depicted in the hierarchy are not directly usable but
-  can be inherited from and extended if necessary in order to meet the
-  requirements of testing code.
+  can be inherited from and extended if necessary to meet the
+  requirements of the testing code.
```

Comment:
I think this would be better phrased '...if necessary for the purposes of testing'